test/fixtures/gen_ali.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
```

Comment:
Does some kind of test program go with this that you haven't checked in?

Reply:
The following is how I run the test:

```bash
$ cd snowfall/test/fixtures
$ ./gen_ali.py
# It generates: hyp.pt, ref.pt, sym.txt

$ snowfall ali edit-distance  -r ./ref.pt -h ./hyp.pt -t phone -o a.txt -s ./sym.txt
# It generates the output file: a.txt
# I checked manually that `a.txt` contains the expected result.
```

---

The files in `snowfall/bin` contain simple wrappers around the existing code. So I don't write unit tests
for command-line tools. Also, snowfall currently contains only a few unit tests.

Reply:
Ah, cool.  Perhaps can note that down in a file checked in somewhere?  Like a README.md somehwere in the cli directory, if there is one?

Reply:
.. and is it possible to run that 'snowfall' command without using setup.py to install?

Reply:
> .. and is it possible to run that 'snowfall' command without using setup.py to install?

Yes, it is possible.

There are two ways.
(1)
```
cd snowfall
python3 ./snowfall/bin/snowfall
```


(2)
```
chmod +x /path/to/snowfall/snowfall/bin/snowfall
export PATH=/path/to/snowfall/snowfall/bin:$PATH
# then you can run
snowfall --help
```

The following is my setting (without installing k2, snowfall, and lhotse)

```bash
export PYTHONPATH=$HOME/open-source/k2/build_release/lib:$PYTHONPATH
export PYTHONPATH=$HOME/open-source/k2/k2/python:$PYTHONPATH
export PYTHONPATH=$HOME/open-source/snowfall:$PYTHONPATH
export PYTHONPATH=$HOME/open-source/lhotse:$PYTHONPATH
export PATH=$HOME/open-source/snowfall/snowfall/bin:$PATH
```

Reply:
I've added a file `snowfall/bin/README.rst`. Please see
https://github.com/k2-fsa/snowfall/blob/fcf0c53c555b89f8fff36d085f9a121c228b90d5/snowfall/bin/README.rst